src/components/Avatar/Avatar.stories.tsx

```diff
@@ -4,7 +4,10 @@ import { getStoryAssetPath } from 'src/services';
 import { Vector3 } from 'three';
 import { FileDropper } from 'src/components/FileDropper/FileDropper.component';
 import { environmentPresets } from 'src/services/EnvironmentMap.service';
-import { Avatar, CAMERA } from './Avatar.component';
+import { Avatar as AvatarWrapper, CAMERA } from './index';
```

Comment:
Let's not expose code snippets as `<AvatarWrapper ... />`.
Instead
```typescript
import { Avatar, CAMERA } from 'src/components/Avatar';
```

Reply:
we add a storybook alias to Avatar, we don't expose AvatarWrapper